app/views/migration/_form.html.erb

```diff
@@ -57,7 +57,7 @@
             <%= wr.input(:dc_type) %>
             <%= wr.input(:dcterms_created, as: :date, html5: true, input_html: date_format_fallback_attributes) %>
             <%= wr.input(:dc_creator) %>
-            <%= wr.input(:edm_rights_id, as: :radio_buttons, collection: edm_rights_options, wrapper_html: { 'data-error-msg-key': 'contribute.campaigns.migration.form.validation.web-resource-license' }) %>
+            <%= wr.input(:edm_rights_id, as: :radio_buttons, collection: edm_rights_options, item_wrapper_class: 'license-radio-option', wrapper_class: 'license-radios input', wrapper_html: { 'data-error-msg-key': 'contribute.campaigns.migration.form.validation.web-resource-license' }) %>
```

Comment:
Is it intended to overwrite the wrapper class?

If not, instead add `class: 'license-radios'` to `wrapper_html` and the class will be _added_ to the default ones.

Reply:
I've made htis change as it's slightly shorter, but the effect was to change this:
```
"license-radios input radio_buttons optional contribution_ore_aggregation_edm_isShownBy_edm_rights_id"
```
to (now) this:

```
"input radio_buttons optional contribution_ore_aggregation_edm_isShownBy_edm_rights_id license-radios"
```